database/src/lib.rs

```diff
@@ -82,45 +78,66 @@
 //!
 //! The default is `heed`.
 //!
+//! `tracing` is always enabled and cannot be disable via feature-flag.
+//! <!-- FIXME: tracing should be behind a feature flag -->
```

Comment:
Am willing to do feature gated `tracing`, although I'm not how sure that will work with stuff like `#[instrument]`.

Reply:
I am ok with just leaving this as a TODO for now